src/views/Albums.vue

```diff
@@ -30,30 +30,31 @@
 	</EmptyContent>
 
 	<!-- Folder content -->
-	<Grid v-else-if="!loading">
+	<div v-else-if="!loading">
 		<Navigation v-if="folder"
 			key="navigation"
 			v-bind="folder"
 			:root-title="rootTitle"
 			:show-actions="true" />
-
-		<!-- Empty folder, should only happen via direct link -->
-		<EmptyContent v-if="isEmpty" key="emptycontent" illustration-name="empty">
-			{{ t('photos', 'No photos in here') }}
-		</EmptyContent>
-
-		<!-- Folders and files list -->
-		<template v-else>
-			<Folder v-for="dir in folderList"
-				:key="dir.fileid"
-				v-bind="dir"
-				:show-shared="showShared" />
-			<File v-for="file in fileList"
-				:key="file.fileid"
-				:list="fileList"
-				v-bind="file" />
-		</template>
-	</Grid>
+		<Grid>
```

Comment:
I feel like this is not related to this pr, no?
What is this trying to fix?

Reply:
I changed rendering of navigation to fit the new Virtual Grid but it was impacting the other views so I had to make a quick fix there also.

If we consider to add the virtual grid everywhere then it's fine!